tests/expressions/tests.py

```diff
@@ -1075,6 +1095,43 @@ def test_update_UUIDField_using_Value(self):
         UUID.objects.update(uuid=Value(uuid.UUID('12345678901234567890123456789012'), output_field=UUIDField()))
         self.assertEqual(UUID.objects.get().uuid, uuid.UUID('12345678901234567890123456789012'))
 
+    def test_deconstruct_Value(self):
+        value = Value('name')
+
```

Comment:
The same suggestion about removing the blank lines applies to all the tests in this PR.